osu!taiko World Cup 2019 Results Newspost (#2279)
ppy/osu-wiki: news/2019-05-27-osutaiko-world-cup-2019-concludes.md

---
layout: post
title: "osu!taiko World Cup 2019 Concludes"
date: 2019-05-27 22:00:00 +0000
---
 

> *sniff sniff...* do you smell that? Yeah, smelt like ...*sniff*... poppin' gamers in here. What's poppin' gamers? The osu!taiko World Cup just came to a close last weekend - read on to find out what happened in the Grand Finale!

Reviewer: ```suggestion
*sniff sniff...* do you smell that? Yeah, smells like... *sniff...* poppin' gamers in here. What's poppin' gamers? The osu!taiko World Cup just came to a close last weekend! Read on to find out what happened in the grand finale!
```

 
![](/wiki/shared/news/banners/TWC_2019_banner.jpg)
 
Six weeks ago, the osu!taiko World Cup stepped boldly into the limelight, returning for its 9th instalment. 

> We've come a long way since our inception in 2011 — long gone are the days where multiple country teams run rampant, with dominating performances from the likes of both Japan Team A and B, and curious oddities such as the great Italy-Netherlands Alliance of 2011-2013.

Reviewer: ```suggestion
We've come a long way since our inception in 2011—long gone are the days where multiple country teams run rampant, with dominating performances from the likes of both Japan Team A and B, and curious oddities such as the great Italy-Netherlands Alliance of 2011-2013.
```

 
This year is special in the sense that it is the first time the osu!taiko World Cup has had 32 unique countries participating in a battle for the very top. 

> With an upgrade to match sizes and the introduction of a terrifying Grand Finals pool, this year undoubtedly saw some of the greatest moments in TWC history — a momentous occasion and a cause for celebration for all osu!taiko fans and supporters alike.

Reviewer: ```suggestion
With an upgrade to match sizes and the introduction of a terrifying Grand Finals pool, this year undoubtedly saw some of the greatest moments in TWC history—a momentous occasion and a cause for celebration for all osu!taiko fans and supporters alike.
```

 
Without further ado, let's get right into the results!
 

> [![](/wiki/shared/news/2019-05-27-osutaiko-world-cup-2019-concludes/podium.jpg)](https://osu.ppy.sh/help/wiki/Tournaments/TWC/2019#finals.1)

Reviewer: the link goes to wrong place, grand finals aren't on the wiki yet

Reviewer: should be fine if this is just a link to the wiki page, though. match results is kinda an odd place for this to lead
```suggestion
[![](/wiki/shared/news/2019-05-27-osutaiko-world-cup-2019-concludes/podium.jpg)](https://osu.ppy.sh/help/wiki/Tournaments/TWC/2019)
```

**Congratulations to Team Japan for winning the osu!taiko World Cup 2019!**
 

> With new leadership in the form of [iceOC](https://osu.ppy.sh/users/5482401), Team Japan showed few signs of weakness throughout the tournament, and will continue to defend their championship title for the **sixth** year in a row! Surely there must be someone that can arrest their momentum...right? ...hello?

Reviewer: ```suggestion
With new leadership in the form of [iceOC](https://osu.ppy.sh/users/5482401), Team Japan showed few signs of weakness throughout the tournament, and will continue to defend their championship title for the **sixth** year in a row! Surely there must be someone that can arrest their momentum... right? hello?
```

> Team Japan will receive the grand prize: $150 per team member, profile badge, and the highly coveted "osu!taiko Champion" user title for one year!

Reviewer: ```suggestion
Team Japan will receive the grand prize: $150 per team member, profile badges, and the highly coveted "osu!taiko Champion" user title for one year!
```

 
**Team Taiwan will continue to defend their silver title for the fifth year running!**
 

> Despite losing their powerhouse [stu00608](https://osu.ppy.sh/users/1977606) from last year, Taiwan's tenacity remains unchallenged in the face of any opposition. Veteran Team Captain [\-\[ ix Ishida xi \]\-](https://osu.ppy.sh/users/242910) (in her 9 year tenure) led her team with flair and elegance, clutching second place by a hair - a splendid performance indeed.

Reviewer: ```suggestion
Despite losing their powerhouse [stu00608](https://osu.ppy.sh/users/1977606) from last year, Taiwan's tenacity remains unchallenged in the face of any opposition. Veteran Team Captain [-[ ix Ishida xi ]-](https://osu.ppy.sh/users/242910) (in her 9 year tenure) led her team with flair and elegance, clutching second place by a hair—a splendid performance indeed.
```

> Team Taiwan will receive a prize of $80 per team member, and a osu!taiko World Cup profile badge to commemorate their success.

Reviewer: ```suggestion
Team Taiwan will receive a prize of $80 per team member and profile badges to commemorate their success.
```

> **Team South Korea makes their debut appearance on the TWC podium, taking home the bronze!**

Reviewer: ```suggestion
**Team South Korea makes their debut on the TWC podium, taking home the bronze!**
```

Author: debut appearance is also correct here i believe, stylistic choice

Reviewer: sounds like a tautology to me ¯\\_(ツ)_/¯

 
With an expanded roster compared to last year, Team Captain [Peaceful](https://osu.ppy.sh/users/165027) led his team in a bloody crusade, laying waste to many teams in a number of surprising victories. Their performance this year was absolutely mind-boggling and with this victory, they become the second country ever to have stood on the podium across all the World Cups!
 

> Team South Korea will receive a prize of $40 per team member, and a osu!taiko World Cup profile badge to commemorate their success.

Reviewer: ```suggestion
Team South Korea will receive a prize of $40 per team member and profile badges to commemorate their success.
```

 
--------
 
I'd like to take this opportunity to thank all the players for their excellent sportsmanship, and the tournament staff for their undying commitment to making things run smoothly. I'd also like to thank the viewers for showing their support for the teams and creating a truly wonderful environment for this competition.
 

> Finally, on a more personal note, I'd like to thank my fellow mappool selectors for all the wonderful moments we had together - working with you to to shape the direction of this tournament has been my pleasure, and I look forward to seeing what the future of the osu!taiko World Cup has in store.

Reviewer: ```suggestion
Finally, on a more personal note, I'd like to thank my fellow mappool selectors for all the wonderful moments we had together—working with you to to shape the direction of this tournament has been my pleasure, and I look forward to seeing what the future of the osu!taiko World Cup has in store.
```
